pants.ini

```diff
@@ -83,6 +83,17 @@ pants_ignore: +[
     '/build-support/bin/native/src',
   ]
 
+remote_execution_server: remotebuildexecution.googleapis.com
+remote_store_server: remotebuildexecution.googleapis.com
+# This file might not exist on your machine. If this default fails, run `find /usr -name '*.pem'`
+# and override this value via the env var PANTS_REMOTE_CA_CERTS_PATH.
+remote_ca_certs_path: /usr/local/etc/openssl/cert.pem
+remote_instance_name: projects/pants-remoting-beta/instances/default_instance
+remote_execution_extra_platform_properties: [
+    # This allows network requests, e.g. to resolve dependencies with Pex.
+    "dockerNetwork=standard",
+    "container-image=docker://marketplace.gcr.io/google/rbe-ubuntu16-04@sha256:da0f21c71abce3bbb92c3a0c44c3737f007a82b60f8bd2930abc55fe64fc2729",
```

Comment:
Stu and I discussed instead using our custom Centos7 image. For now, we'll stick with this because it's Ubuntu 16 so closer to what we have in CI. It's possible that we will need to revisit this sometime soon.

Reply:
We should use whatever is closest to our travis environment probably... the Centos6/7 would primarily be relevant to wheel building, but our unit tests successfully run PEX resolves directly in the travis environment currently, and it would probably reduce complexity a bit to not change the platform we're running unit/integration tests on right now.